.babelrc

```diff
@@ -12,8 +17,14 @@
   "env": {
     "production": {
       "plugins": [
-        "transform-react-remove-prop-types"
+        [
+          "transform-react-remove-prop-types",
+          {
+            "mode": "remove",
```

Comment:
Here as well. Any particular reason for doing it?

Reply:
`"mode": "remove"` together with `"removeImports": true` on the next line configures the plugin in addition to removing constructs `Component.propTypes={...}` to also remove `import PropTypes from 'prop-types'`. Shaves off another couple lines per Component.